src/Input/Textarea/TextareaStyle.ts

```diff
@@ -42,34 +33,24 @@ export const TextareaLabel = styled.label<TextareaLabelProps>`
 interface TextareaLabelProps {
   floating: boolean;
   status?: string;
+  small?: boolean;
 }
 
 export const TextareaInput = styled.textarea<TextareaInputProps>`
   position: relative;
   width: 100%;
   outline: none;
-  font-size: 1.1em;
+  font-size: 16px;
   line-height: 1.5;
-  padding: 15px 20px;
-  border: ${({ status }) =>
-    status === 'error'
-      ? `2px solid ${PrimaryColor.glintsred}`
-      : `2px solid ${Greyscale.grey}`};
-  transition: border .5s;
+  padding: ${({ small }) => (small ? '8px 16px' : '12px 16px')};
+  background-color: ${({ status }) =>
+    status === 'error' ? ' rgba(236, 39, 43, 0.08)' : 'rgb(1, 126, 183, 0.08)'};
+  border: unset;
```

Comment:
Is this important? Why?

Reply:
Yes, required.
Because the input field already has it's own default border. 
For our component as we have kind of a box with background colour the border isn't required.